distribution/docs/src/main/resources/content/_managing/_running/solr-commands.adoc

```diff
@@ -0,0 +1,28 @@
+:title: Solr Commands
+:type: subMaintaining
+:status: published
+:parent: Available Console Commands
+:summary: Solr commands available.
+:order: 00
+
+== {title}
+
+.[[_solr_command_descriptions]]Solr Command Descriptions
+[cols="1m,9a" options="header"]
+|===
+
+|Command
+|Description
+
+|solr:backup
+|Creates a backup of the selected Solr core/collection. This uses the Solr interface for creating
+the backup. In Solr Cloud deployments the selected backup directory must exist on all Solr nodes.
```

Comment:
Does the selected backup directory need to also be shared between them (and not just exist on each node)?

Reply:
Added a note that it needs to be shared for it to work.